public/docs/_examples/toh-5/ts/app/app.module.1.ts

```diff
@@ -4,7 +4,7 @@ import { BrowserModule }  from '@angular/platform-browser';
 import { FormsModule }    from '@angular/forms';
 
 import { AppComponent }   from './app.component';
-
+import { HeroDetailComponent }  from './hero-detail.component';
 import { HeroesComponent }  from './heroes.component';
 
```

Comment:
One more blank line to drop here (since you'll be fixing the indentation in the .jade file anyways :).